stable/rabbitmq/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "{{ template "rabbitmq.fullname" . }}"
+  labels:
+    app: {{ template "rabbitmq.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
instead of using `"{{ .Chart.Name }}-{{ .Chart.Version }}"` you should create the following template and use it here:

`chart: {{ template "rabbitmq.chart" . }}`

Template:

```
{{/*
Create chart name and version as used by the chart label.
*/}}
{{- define "rabbitmq.chart" -}}
{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
{{- end -}}
```

This change in the labels applies to all files

Reply:
mmm,seems over kill - is there a guideline about when to do this?

Reply:
done